Rubberduck.VBEEditor/SafeComWrappers/VBA/VBComponents.cs

```diff
@@ -152,6 +152,9 @@ public void ImportSourceFile(string path)
 
         public void RemoveSafely(IVBComponent component)
         {
+            if (component == null)
+                { return; }
```

Comment:
Again, this leverages the `==` operator override; works but `if (component.IsWrappingNullReference) { return; }` would be better I think.

Reply:
Please either remove the extraneous indentation, or bring `{ return; }` on the same line as the `if`.